examples/backup-github.mjs

```diff
@@ -25,11 +25,14 @@ if (process.env[token]) {
     Authorization: `token ${process.env[token]}`
   }
 }
-let res = await fetch(`https://api.github.com/users/${username}/repos`, {headers})
+let res = await fetch(`https://api.github.com/users/${username}/repos?per_page=1000`, {headers})
 let data = await res.json()
-let urls = data.map(x => x.git_url)
+let urls = data.map(x => x.git_url.replace('git://github.com/', 'git@github.com:'))
 
 await $`mkdir -p backups`
 cd('./backups')
 
-await Promise.all(urls.map(url => $`git clone ${url}`))
+await urls.reduce(async (m, url, i) => {
```

Comment:
Maybe chunk array and wait in batches?

Reply:
I hit the rate limit, so single thread is more reliable.

Reply:
Well, it’s just for learning purposes.

Reply:
Also if per command - for loop is more readable.

Reply:
git clone is i/o in most and doesn't really benefit from 100 threads

Reply:
git clone is i/o in most and doesn't really benefit from 100 threads

Reply:
`reduce → for of`, ok

Reply:
Will be cool to find an example to show await Promise.all with [].map usage.